src/Pickles/Pickles/Runner.cs

```diff
@@ -97,23 +97,38 @@ private static void SetResultsForIndividualScenariosUnderFeature(FeatureNode fea
 
                 if (scenario != null)
                 {
-                    featureElement.Result = testResults.GetScenarioResult(scenario);
+                    try { 
+                        featureElement.Result = testResults.GetScenarioResult(scenario);
+                    }
+                    catch (Exception e)
+                    {
+                        Log.Error("An exception occured in aligning results with the feature for : " +
```

Comment:
You're blindly logging all exceptions but not taking appropriate action. Not good.